lca-main.yaml

```diff
@@ -18,6 +18,10 @@ Transform: AWS::Serverless-2016-10-31
 Description: Amazon Transcribe Live Call Analytics with Agent Assist - LCA (v0.7.1) (SO9072)
 
 Parameters:
+  ExpirationInDaysForIASTACK:
+    Type: Number
+    Description: "Number of days after which bucket objects will be deleted."
```

Comment:
Add clarification: Number of days after which bucket objects will be deleted *from the Recordings bucket*